src/pages.c

```diff
@@ -21,6 +21,13 @@
 #else
 #define PAGES_FD_TAG -1
 #endif
+#ifdef JEMALLOC_HAVE_PRCTL
+#include <sys/prctl.h>
+#ifndef PR_SET_VMA
+#define PR_SET_VMA 0x53564d41
+#define PR_SET_VMA_ANON_NAME 0
+#endif
```

Comment:
If `PR_SET_VMA` is undefined, doesn't that mean that the kernel doesn't support it? Maybe it should be a configuration error if `PR_SET_VMA` is undefined, but `enable-pageid` is `yes`?

Reply:
at worse it s a no-op, in some cases the kernel supports it but not defined in the header (my case).

Reply:
But then `os_page_id` will return `-1` and fail the assertion you added.

Reply:
ah true :-)